catalog/ui/catalog-ui-search/src/main/webapp/component/query-status/query-status-header.hbs

```diff
@@ -13,25 +13,13 @@
  --}}
 <th>
     Source
-    <div class="column-absolute">
-    Source
-    </div>
 </th>
-<th>
+<th data-help="This is the number of results that made it into the Top.">
```

Comment:
I like the idea of providing that breadcrumb of why there, so I think we can do something like what you're suggesting while also mentioning sorting:
"This is the number of results returned based on current sorting relevance."